src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
```suggestion
  hide() {
    this.hidden = !this.hidden;
```

This method should work in such a way: if 'hidden' is 'true', then calling this method should swap it to 'false' and vice versa. You can use operator '!' for this purpose